test/interfaces.jl

```diff
@@ -0,0 +1,113 @@
+#using QEDprocesses
+using Random
+using Distributions
+using QEDbase
+
+RNG = MersenneTwister(708583836976)
+DIM = rand(RNG, 2:8)
+
+ATOL = 0.0
+RTOL = sqrt(eps())
+
+RAND_MAX = rand(RNG)
+RAND_EXACTNESS = rand(RNG, Bool)
+
+struct TestSetup{D} <: QEDevents.AbstractComputationSetup
+    dist::D
+end
+
+function TestSetup(dim::Integer)
+    return TestSetup(product_distribution(Uniform.(-rand(dim), 1.0)))
+end
+
+Base.size(stp::TestSetup, N::Integer) = size(stp.dist)[1] ###best way?
```

Comment:
Better use `size(stp.dist,1)`.

Reply:
That doesn't work:
```julia
MethodError: no method matching size(::Product{Continuous, Uniform{Float64}, Vector{Uniform{Float64}}}, ::Int64)
```

Reply:
Ah ok, this is the weird implementation of `size` in `Distributions.jl`. Then ignore my comment.